packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2474,7 +2594,8 @@ class InputDecoration {
   /// If null, defaults to the [helperStyle].
   final TextStyle counterStyle;
 
-  /// If true the decoration's container is filled with [fillColor].
+  /// If true the decoration's container is filled with [fillColor] when not
```

Comment:
Is the hover color not involved here as well?

Reply:
Yes.  Added.